### ISSUE TITLE
deploy prodigy to production environment

### ISSUE THREAD

**Issue by jerielizabeth** [Princeton-CDH/ppa-nlp#49]
- [x] configure a descriptive name for the test poem set for storage in prodigy
- [x] confirm that data can be exported

**Comment by rlskoeser** [comment]
This is in progress. We have revised the prosody configuration setup so we can more easily customize the prosody configuration when we make other changes related to recipe or input dataset. Both @laurejt and I are able to run the ansible deploys and we've run on both production and staging (production deploy in progress due to a couple of snags).

We're having problems with the tigerdata mount point - trying to read is timing out, and this was causing nginx to crash in staging (since nginx is trying to serve out images from that mount point). I confirmed that by temporarily disabling the image path in the nginx config, and then prodigy behaved much better.

We've got a better dataset name and exported some test datasets in staging to make sure we're comfortable with the information included in the output.
